src/nxt_runtime.c

```diff
@@ -789,6 +792,16 @@ nxt_runtime_conf_init(nxt_task_t *task, nxt_runtime_t *rt)
     rt->state = NXT_STATEDIR;
     rt->control = NXT_CONTROL_SOCK;
     rt->tmp = NXT_TMPDIR;
+    rt->conf_gen = 0;
+
+    time(&rawtime);
+    timeinfo = gmtime(&rawtime);  //convert to UTC
+    strftime((char*)buffer, 25, "%Y-%m-%dT%H:%M:%S.000Z", timeinfo);
+
+    rt->conf_ltime.length = strlen((char*)buffer);
+    rt->conf_ltime.start = (u_char*) malloc(rt->conf_ltime.length + 1);
```

Comment:
Diit malloc(3).

Hmm, duplicated code...

Reply:
@ac000 How I can fix code duplicate? I'm using this code to get time and convert it to nxt_str_t. I can create nxt_time.c with function to reuse it
```c
time_t         rawtime;
u_char         buffer[25];
struct         tm *timeinfo;

time(&rawtime);
timeinfo = gmtime(&rawtime);  //convert to UTC
strftime((char*)buffer, 25, "%Y-%m-%dT%H:%M:%S.000Z", timeinfo);

rt->conf_ltime.length = strlen((char*)buffer);
rt->conf_ltime.start = nxt_malloc(rt->conf_ltime.length + 1);
```